test/corpus/errors.txt

```diff
@@ -0,0 +1,44 @@
+=================================================
+Error detected at globally reserved keyword
+=================================================
+
+<?php
+$object->
+class Test {  // 'class' should not be treated as a property
+}
+
+class Example {
+    public function test() {
+        $this->    // incomplete
+        function process() {}  // function should be recognized
```

Comment:
both `class` and `function` can technically be properties here so not sure if this will work